### DISCUSSION TITLE
No Tag to Display Shadow For Vxl-based Vehicles' Turret

### DISCUSSION THREAD

**Discussion by Daiki9527** [Phobos-developers/Phobos#1035] [Ideas and feedback]
### First check

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

### Description

Corrently,There Is No Tag to Display Shadow For Vxl-based Vehicles' Turret, specially when it comes to jumpjet."UseTurretShadow=" only render the turret's shadow for shp-based units.

### INI code

```ini
[UnitTypesArt]
VxlUseTurretShadow=yes/no
TurretShadowIndex=1

0 For **tur.vxl
1 For **tur1.vxl
5 For **tur5.vxl
```

**Comment by McPwny** [comment]
turret shadows actually do not render for .shp based vehicles at all. its a vanilla bug

**Comment by Super-StarX** [comment]
see #868
